tests/tests.py

```diff
@@ -56,7 +57,22 @@ def _get_form_str(self, form_str):
         return form_str
 
     def assert_form_renders(self, form, html):
-        self.assertHTMLEqual(str(form), self._get_form_str(html))
+        # Django causes a DeprecationWarning on Python 3.3, 3.4
+        if (3, 3) >= sys.version_info > (3, 5):
```

Comment:
Good catch. Which means if the DeprecationWarnings are not appearing, that particular issue must have gone away... I'll look in to it.